docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,37 @@
+<style type="text/css">
+  .tg  {border-collapse:collapse;border-spacing:0;}
+  .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg .tg-c3ow{border-color:black;text-align:center;vertical-align:top}
+  .tg .tg-0pky{border-color:black;text-align:left;vertical-align:top}
+  .tg .tg-dgl5{border-color:black;background-color:#FFF;text-align:left;vertical-align:top}
+  </style>
+  <table class="tg">
+    <tr>
```

Comment:
This row should go inside a `<thead>` element, and the rest should go inside a `<tbody>`